services/jsdelivr/jsdelivr-base.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Joi = require('joi')
+const { metric } = require('../../lib/text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  total: Joi.number().required(),
+}).required()
+
+const keywords = ['jsDelivr', 'hits']
+
+const periodMap = {
+  hd: 'day',
+  hw: 'week',
+  hm: 'month',
+  hy: 'year',
+}
+
+class BaseJsDelivrService extends BaseJsonService {
+  static render({ period, hits }) {
+    return {
+      message: `${metric(hits)} hits/${periodMap[period]}`,
+      //color: downloadCount(hits),
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'jsDelivr',
```

Comment:
The left-hand side of the badge should be all lowercase: https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines (even if the service usually styles their name in mixed-case)